### PR TITLE
Set up Actions

### PR DESCRIPTION
**Pull request by josh**

### CODE REVIEW

**Comment by josh** [comment]
@pmarsceill could you give my workflow file a review?

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -0,0 +1,25 @@
+name: ShellCheck
+
+on: [push]
+
+jobs:
+  lint:
```
Comment (josh) [inline .github/workflows/main.yml:6]: @pmarsceill what's a best practice for naming the job?

File: .github/workflows/main.yml [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v1
+
+    - name: Install ShellCheck
+      run: sudo apt-get install shellcheck
```
Comment (josh) [inline .github/workflows/main.yml:13]: Is this the best practice for installing packages?

File: .github/workflows/main.yml (hunk 1 above)
Comment (pmarsceill) [reply .github/workflows/main.yml:6]: This is just the ID that the system uses. You can optionally include a `name:` definition that is more user-friendly to show up in the UI. Jobs will be shown as `Workflow name / Job name` so the best practice is to make it not look like garbage by using super long strings.

File: .github/workflows/main.yml (hunk 2 above)
Comment (pmarsceill) [reply .github/workflows/main.yml:13]: Seems to be? If it works and is performant enough for you, I'd say 👍

File: .github/workflows/main.yml (hunk 1 above)
Comment (pmarsceill) [reply .github/workflows/main.yml:6]: This is how it currently shows up:

image


If you cared, you could do:

```
jobs:
  lint:
    name: Lint
```

**Pull request merged**